project_tests.sh

```diff
@@ -3,10 +3,10 @@ set -e
 
 rm -f build/*.xml
 
-export SYMFONY__FEATURE_CAN_ALWAYS_USE_HYPOTHESIS__OVERRIDE=false
+export SYMFONY_ENV=test
```

Comment:
This should mean `bin/console security:check` works here. It probably should use the `test` environment? (Doesn't actually interact with anything.)

`vendor/bin/security-checker security:check` actually bypasses the app too.